.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015", "stage-0"]
+}
```

Comment:
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline